compact/tracking/definitions_craterlake.xml

```diff
@@ -48,7 +48,7 @@
     <comment> Forward TOF region  </comment>
     <constant name="ForwardTOFRegion_zmin"        value="174.0*cm" />
     <constant name="ForwardTOFRegion_tan"         value="CentralTrackingRegionP_tan" />
-    <constant name="ForwardTOFRegion_minR"        value="8*cm" />
+    <constant name="ForwardTOFRegion_minR"        value="10*cm" />
```

Comment:
As I wrote, we cannot fit the actual beam pipe in the current detector. Please let me know the procedure to get it approved. Do you know who should be contacted?

Reply:
The current procedure is to send an email with requests for changes to the detector parameter table to the detector parameter table maintainers, the project's experiment contacts, collaboration leadership, and the specific detector contacts. I will send you their emails instead of posting them here.